Numpy_task/numpy_challenge.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+
+
+def multiplication_check(matrices_list):
+    """ Input: a list of 2-dimensional matrices.
+        Return: True if they can be multiplied by each other in the order in which they are in the list,
+        and False if they cannot be multiplied.
+    """
+    result = True
+    multiplication_result_shape = matrices_list[0].shape
+    for i in range(1, len(matrices_list)):
+        if multiplication_result_shape[1] != matrices_list[i].shape[0]:
+            result = False
+            break
+        else:
+            multiplication_result_shape = (multiplication_result_shape[0], matrices_list[i].shape[1])
+    return result
```

Comment:
Норм, можно немного упростить
```suggestion
def multiplication_check(matrices_list):
    """ Input: a list of 2-dimensional matrices.
        Return: True if they can be multiplied by each other in the order in which they are in the list,
        and False if they cannot be multiplied.
    """
    multiplication_result_shape = matrices_list[0].shape
    for i in range(1, len(matrices_list)):
        if multiplication_result_shape[1] != matrices_list[i].shape[0]:
            return False
        else:
            multiplication_result_shape = (multiplication_result_shape[0], matrices_list[i].shape[1])
    return True
```